Sources/Controllers/MessagesViewController+Keyboard.swift

```diff
@@ -30,12 +30,14 @@ extension MessagesViewController {
 
     internal func addKeyboardObservers() {
         NotificationCenter.default.addObserver(self, selector: #selector(MessagesViewController.handleKeyboardDidChangeState(_:)), name: .UIKeyboardWillChangeFrame, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(MessagesViewController.handleKeyboardDidChangeState(_:)), name: .UIKeyboardDidChangeFrame, object: nil)
```

Comment:
Note: only listening to UIKeyboardWillChangeFrame will cause issues in certain situations where keyboard was in the process to hide/show and then we get into our msgCollectionView, then we missed the willChange event and didn't catch up with didChange event